ArcGISRuntimeSDKQt_CppSamples/Layers/DisplayAnnotation/README.md

```diff
@@ -0,0 +1,37 @@
+# Display Annotation
+
+Display annotation from a feature service URL.
+![](screenshot.png)
```

Comment:
screenshot should be 500x500 pixels. We also generally don't include the full application window so that it is more platform agnostic in the image. There are some details about the screenshot in the wiki I sent you on Slack